content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -75,10 +75,13 @@ An _output entry_ (`outs`) consists of these fields:
 - `nfiles`: If a directory, number of files inside.
 - `cache`: Whether or not this file or directory is <abbr>cached</abbr> (`true`
   by default, if not present). See the `--no-commit` option of `dvc add`.
+- `persist`: Whether the output file/dir should remain in place while
+  `dvc repro` runs. By default outputs are deleted when `dvc repro` starts (if
+  this value is not present).
```

Comment:
All the other changes to this doc are unrelated, sorry. They're just copy edits along-the-way.

Reply:
please, please, let's not do this :)

Reply:
OK